### DISCUSSION TITLE
GitHub issue responds with a status of 500

### DISCUSSION THREAD

**Discussion by boghyon** [community/community#23162] [Projects and Issues]
<p>People are not able to access particularly this issue: <a href="https://github.com/OData/ODataSamples/issues/29" class="inline-onebox" rel="noopener nofollow ugc">Sample services don't support CORS, which makes JS clients hard to consume · Issue #29 · OData/ODataSamples · GitHub</a> <strong>since today</strong>.</p>
<p>The above issue is linked from <a href="https://github.com/SAP/openui5/issues/3150" class="inline-onebox" rel="noopener nofollow ugc">v4.ODataModel evaluates preflight requests · Issue #3150 · SAP/openui5 · GitHub</a></p>
<blockquote>
<p>unfortunately I am not able to see the issue OData/ODataSamples#29. I get a code 500 response (only for this issue).</p>
</blockquote>
<p>I’m not able to access the issue either.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/c/4c5d64a12cd165cfa959a5ef4d37578fbd6daafa.jpeg" data-download-href="/uploads/short-url/aTynExAnSzESdiZGYATHTIKo9wS.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/12143247/181075395-ea631f33-f84d-4167-ae15-fb0ae03ef4b1.jpeg" alt="image" data-base62-sha1="aTynExAnSzESdiZGYATHTIKo9wS" width="690" height="367" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c5d64a12cd165cfa959a5ef4d37578fbd6daafa_2_690x367.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c5d64a12cd165cfa959a5ef4d37578fbd6daafa_2_1035x550.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c5d64a12cd165cfa959a5ef4d37578fbd6daafa_2_1380x734.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/c/4c5d64a12cd165cfa959a5ef4d37578fbd6daafa_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1652×881 258 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by rmuchall** [comment]
<p>I’m experiencing the same problem with this issue:<br>
<a href="https://github.com/electron/electron/issues/2867" class="onebox" target="_blank" rel="noopener nofollow ugc">https://github.com/electron/electron/issues/2867</a></p>

**Comment by boghyon** [comment, marked as answer]
<p>I’m able to access all mentioned issues now.</p>
